src/components/SingletonEditor.test.tsx

```diff
@@ -29,84 +27,45 @@ const defaultAppServices = {
 const noop = () => {};
 
 describe(Editor.name, () => {
-  it("should hide EmailEditor using the default Context", () => {
```

Comment:
I removed this first test because the new one tests all the scenarios:

* After starting the application with the initial state
* After loading a design
* After unloading the design